setup.py

```diff
@@ -0,0 +1,16 @@
+from setuptools import setup
+
+
+setup(
+    name='pa11y_runner',
+    version='0.1',
+    py_modules=['pa11y_runner'],
+    install_requires=[
+        'Click',
```

Comment:
It might be overkill, but do you want/need to set lower bounds on these requirements?

Reply:
good idea.  I pinned the versions in the requirements.txt, but forgot when I created the setup.  Because I don't know if the dependencies follow SemVer, I think it's best to pin to a specific version that I know is working.  That also has the advantage of always knowing when a dependency has been updated rather than having the project break because the dependency broke backwards compatibility.

Reply:
This is always a tough one. If we don't know that they follow semver there is not much you can do.  Someone is always going to be unhappy and there is always the potential for breakage.  Having said that, I think you should use the compatible release version string anyway :) . Something like `click~=7.1.2`, which I *think* allows anything in the 7.1 release range.

Reply:
Without knowing that Click follows SemVer, we have no way of knowing if 7.1.3 is compatible with 7.1.2.  But since I used this with 7.1.2, I can guarantee that it works with that version, so I think we're safer pinning to a specific version in this case.  

I'm not a fan of pinning to ranges.  When you build artifacts, you have no way of knowing if a newer version of a dependency has been pulled in.  If you are trying to rebuild a specific release version of an artifact and don't specify the dependency versions to pull in, you cannot guarantee that recreated artifacts are the same as the artifact you are recreating.

I know it's probably not critical in this case, but I'd rather be consistent across all of my projects so that when I'm working on something where it is important, I don't miss it.

Reply:
I think this is okay for a python application, as opposed to a library.  I assume it should be installed in a venv, so its dependencies are isolated anyway.  A library should allow more compatibility since whoever is consuming your library might also want to pull in security fixes of your dependencies, etc and its on them to ensure everything works as expected.